Implement UI header

LukeRyssel opened the pull request:
-- Added button functionality
        -- Changed title
        -- Header things that need to be changed still: Add use for 'get started' button or get rid of it. (Reached out to the client about this)
	modified:   src/components/DesktopContainer.jsx 
	modified:   src/components/HomepageHeading.jsx
	modified:   src/components/MobileContainer.jsx

LukeRyssel committed:
modified:   package-lock.json
	modified:   package.json
	modified:   src/components/DesktopContainer.jsx
	modified:   src/components/HomepageHeading.jsx
	modified:   src/components/MobileContainer.jsx

crobinson42 commented on src/components/DesktopContainer.jsx:
```diff
@@ -16,6 +16,36 @@ import { HomepageHeading } from "./HomepageHeading";
+  };
+  scrollToSpaces = () =>{
+    window.scrollTo({
+      top: 700, 
```
These constant values can change depending on the device height/width. One option is to use anchor tag (ie: `<a href="#spaces" />`) and another option would be to do it programmatically with https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect

crobinson42 replied:
Also, check and see if the `react-router-dom` docs have any support for scrolling to anchor tags within a page.

crobinson42 commented on src/components/DesktopContainer.jsx:
```diff
@@ -16,6 +16,36 @@ import { HomepageHeading } from "./HomepageHeading";
  */
 class DesktopContainer extends Component {
   state = {};
+  
```
These members look like they're reused in the MobileContainer.jsx file - see if you can solve removing the code duplication by extracting the members and making them reusable.

LukeRyssel committed:
modified:   src/HomepageLayout.js -- added IDs to sections
	modified:   src/components/DesktopContainer.jsx -- implemented anchor tags
	modified:   src/components/MobileContainer.jsx -- implemented anchor tags
 Untracked files:
	src/components/Helper.jsx -- added this file to hold resused methods

crobinson42 commented:
@LukeRyssel  I don't see the Helper.js file included in the committed files, am I missing something?

LukeRyssel committed:
new file:   src/components/Helper.jsx

LukeRyssel commented:
> 
> 
> @LukeRyssel I don't see the Helper.js file included in the committed files, am I missing something?

@crobinson42 My bad, I was not aware I had to specify to include new files in the commit. I think you can see it now

crobinson42 commented:
👏 merge it 👌

> On Jul 2, 2021, at 7:06 PM, LukeRyssel ***@***.***> wrote:
> 
> ﻿
> @LukeRyssel I don't see the Helper.js file included in the committed files, am I missing something?
> 
> @crobinson42 My bad, I was not aware I had to specify to include new files in the commit. I think you can see it now
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.
